source/Calamari/Scripts/Octopus.Features.WindowsService_BeforePostDeploy.ps1

```diff
@@ -158,6 +145,16 @@ else
 	}
 }
 
+if ($serviceAccount -eq "_CUSTOM") {
+	# dont use sc.exe to set the username / password, as it may be logged to the windows audit log if process creation event logs are enabled 
+	$wmiService = Get-WmiObject win32_service -filter "name='$serviceName'" -computer "."
+	$result = $wmiService.change($null, $null, $null, $null, $null, $null, $customAccountName, $customAccountPassword, $null, $null, $null)
+	if ($result.ReturnValue -ne "0") {
+		#return codes: https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/change-method-in-class-win32-service#return-value

```

Comment:
Maybe this URL could go into Octopurls and we reference that in the error message below? Not convinced it's too helpful though. 🤔

Reply:
I considered that, or even having a lookup here, but deferred that for the future to see how often we hit it.